deepchem/metrics/__init__.py

```diff
@@ -71,6 +71,9 @@ def pearson_r2_score(y, y_pred):
   """Computes Pearson R^2 (square of Pearson correlation)."""
   return pearsonr(y, y_pred)[0]**2
 
+def r2_score(y, y_pred):
```

Comment:
This function isn't needed (`r2_score` is exported directly from scikit-learn and can be used already)